prov/gni/src/gnix_nic.c

```diff
@@ -50,7 +50,7 @@
 #define GNIX_VC_FL_MIN_SIZE 128
 
 static int gnix_nics_per_ptag[GNI_PTAG_USER_END];
-static DLIST_HEAD(gnix_nic_list);
+DLIST_HEAD(gnix_nic_list);
```

Comment:
Why did this move from static to non-static? If it is being moved from single file scope, to multi-file scope, then are the appropriate protections in place?